extensions/wikia/AdEngine/js/provider/gpt/adDetect.js

```diff
@@ -178,21 +184,21 @@ define('ext.wikia.adEngine.provider.gpt.adDetect', [
 
 		function pollForSuccess() {
 			successTimer = setTimeout(function () {
-				log(['pollForSuccess', slotname], 'info', logGroup);
+				log(['pollForSuccess', slotName], 'info', logGroup);
 				pollForSuccess();
-				findAdInIframe(slotname + ' (poll)', iframe, callAdCallback, noop);
+				findAdInIframe(slotName + ' (poll)', iframe, callAdCallback, noop);
 			}, 500);
 		}
 
 		function msgCallback(data) {
-			log(['msgCallback', slotname, 'caught message', data], 'info', logGroup);
+			log(['msgCallback', slotName, 'caught message', data], 'info', logGroup);
 
 			if (data.status === 'success') {
-				if (expectAsyncSuccess) {
+				if (expectAsyncSuccess || expectAsyncHopWithSlotName) {
```

Comment:
Is here **expectAsyncHopWithSlotName** variable used correctly? Shouldn't be here **expectAsyncSuccessWithSlotName**? If it is correct it seems that expectAsyncSuccessWithSlotName is reduntant to expectAsyncHopWithSlotName